ageing-and-society.csl

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="chicago" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Ageing &amp; Society</title>
+    <id>http://www.zotero.org/styles/ageing-and-society</id>
+    <link href="http://www.zotero.org/styles/ageing-and-society" rel="self"/>
+    <link href="http://www.zotero.org/styles/emu-austral-ornithology" rel="template"/>
+    <link href="https://github.com/citation-style-language/styles/files/1047799/575ac5898d9f13e91d15f5cf-1.pdf" rel="documentation"/>
```

Comment:
@damnation333, I'll just replace this with a link to https://www.cambridge.org/core/journals/ageing-and-society/information/instructions-contributors where this file is accessible.